flow/project.py

```diff
@@ -3169,12 +3383,33 @@ def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
         :type ignore_conditions:
             :py:class:`~.IgnoreConditions`
         :yield:
-            All instances of :class:`~.JobOperation` jobs are eligible for.
+            All instances of :class:`~._JobOperation` jobs are eligible for.
         """
         for job in jobs:
             for op in self._job_operations(job, ignore_conditions):
                 yield op
 
+    @deprecated(deprecated_in="0.10", removed_in="0.13", current_version=__version__)
+    def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
+        """Determine the next eligible operations for jobs.
+
+        :param jobs:
+            The signac job handles.
+        :type job:
+            :class:`~signac.contrib.job.Job`
+        :param ignore_conditions:
+            Specify if pre and/or post conditions check is to be ignored for eligibility check.
+            The default is :py:class:`IgnoreConditions.NONE`.
+        :type ignore_conditions:
+            :py:class:`~.IgnoreConditions`
+        :yield:
+            All instances of :class:`~._JobOperation` jobs are eligible for.
```

Comment:
Here we do return `_JobOperation` objects. I am not sure what the policy here is, @vyasr. We could transform from `_JobOperation` to `JobOperation` but that seems like too much work.

Reply:
I think as long as JobOperation exists and is deprecated but not removed, that transformation is unfortunately probably necessary.

Reply:
Okay, that makes sense. @kidrahahjo It might be easiest to use the existing `_JobOperation` object's data to create corresponding `JobOperation` objects.